app/main/posts/modify/post-media.directive.js

```diff
@@ -16,15 +16,27 @@ function (
     return {
         restrict: 'E',
         replace: true,
-        require: '^^form',
+        require: ['^^form', 'ngModel'],
         scope: {
             mediaId: '=',
             media: '=',
```

Comment:
If you're going to use ngModel it'd be better to remove the `media` input, and maybe `mediaId` too. And handle input/output via the ngModel controller